### PR TITLE
[TSD] Add annotation for ENABLE_ORA_MOBILE_SUPPORT feature toggle

### PR DESCRIPTION
**Pull request by morenol**
**TL;DR -** Add annotations for ENABLE_ORA_MOBILE_SUPPORT feature toggle but does not affect behavior.

**What changed?**

Added annotations for ENABLE_ORA_MOBILE_SUPPORT feature toggle as part of the doc-a-thon event!

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @morenol! I've created [OSPR-5568](https://openedx.atlassian.net/browse/OSPR-5568) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by natabene** [comment]
@morenol Thank you, @robrap FYI.

File: openassessment/xblock/config_mixin.py
Comment (robrap) [inline openassessment/xblock/config_mixin.py:18]: How do you feel about this minor request?
```suggestion
# .. toggle_description: Set to True to enable the ORA2 XBlock to be rendered
```

File: openassessment/xblock/config_mixin.py
Comment (robrap) [inline openassessment/xblock/config_mixin.py:15]: How would you feel about moving these annotations above line 129? If not, above line 26?  It would be nice to move it closer to something that aligns with the `annotation_name`. Line 129 is closer to seeing the default as well.

We could also consider updating line 129 to:
```
return settings.FEATURES.get('ENABLE_ORA_MOBILE_SUPPORT') is True
```
And removing `MOBILE_SUPPORT` altogether. This would make it even more clear that the flag and switch aren't supported for this. It does add a little risk to what would otherwise just be documentation, so I'll leave this to you.

**Review by robrap — COMMENTED** [review]
Thanks @morenol. This is still in draft state, so not sure if you wanted it reviewed yet, but here it is anyway. 😄

File: openassessment/xblock/config_mixin.py
Comment (morenol) [reply openassessment/xblock/config_mixin.py]: Done! You can review this again!

File: openassessment/xblock/config_mixin.py
Comment (robrap) [inline openassessment/xblock/config_mixin.py:21]: @iloveagent57: I think we have this waffle turned on in Stage, although we also have `FEATURES['ENABLE_ORA_MOBILE_SUPPORT']` turned on in Stage.  Do you agree that waffle/switch should not be supported for this toggle, and would you mind deleting the flag `ENABLE_ORA_MOBILE_SUPPORT` in Stage?

File: openassessment/xblock/config_mixin.py
Comment (robrap) [inline openassessment/xblock/config_mixin.py:130]: Sorry I missed this, but consider updating (or following up) with a refactor to declare a SettingDictToggle at the top of this file, and using it here.  See https://edx.readthedocs.io/projects/edx-toggles/en/latest/decisions/0003-django-setting-toggles.html#decision

**Review by robrap — APPROVED** [review]
Looks great. Thank you. I gave one optional refactor that would help provide the location of the setting in the future.

Also - it would be great to hear from @iloveagent57 to confirm that we should be down to just the setting for mobile support.

File: openassessment/xblock/config_mixin.py
Comment (robrap) [reply openassessment/xblock/config_mixin.py:130]: @morenol: If you want to do the SettingDictToggle refactor, we should probably do it on a separate PR so we can land this.  I'm just waiting for confirmation from the owners on making `ENABLE_ORA_MOBILE_SUPPORT` only work with a Django Setting.

Comment (morenol) [reply openassessment/xblock/config_mixin.py:130]: I think that it only can be used through a django setting. The previous attempt to make this use a waffle flag failed because the creation of docker images failed.

 This decorator is evaluated on import time: 
https://github.com/edx/edx-ora2/blob/76269007c4d7157ab62a50c48f2715f90e01dca2/openassessment/xblock/openassessmentblock.py#L495

**Review by robrap — APPROVED** [review]
Thanks again for all the work on this!

**Comment by edx-webhook** [comment]
@morenol 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-ora2&entry.867055334=https://github.com/edx/edx-ora2/pull/1580&entry.1484655318=https://github.com/morenol&entry.752974735=2021-01-27+17:50&entry.1917517419=2021-02-02+22:57&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**
